configuration.rst

```diff
@@ -583,6 +583,12 @@ In addition to your own env vars, this ``.env`` file also contains the env vars
 defined by the third-party packages installed in your application (they are
 added automatically by :ref:`Symfony Flex <symfony-flex>` when installing packages).
 
+.. tip::
+
+    Since the ``.env`` file is read and parsed on every request, you don't need to
+clear the cache on ``dev`` environment nor to restart the (PHP) container in case
+you are using docker.
```

Comment:
Make sure the 2nd and 3rd lines are also indented to the same level as the word `Since`

And let's try this wording:

> clear the Symfony cache or restart the PHP container if you're using Docker.

Reply:
@weaverryan OK, thank you for the review :)